### PR TITLE
Cleanuptests

### PR DESCRIPTION
**Pull request by deculler**
[ X] Wrote test for feature
[ ] Added changes in the Changelog section in README.md
[ ] Bumped version number (delete if unneeded)

**Changes proposed:**
Updates read_table function to use pandas.read_csv rather than pandas.read_table, which was deprecated 0.42

Update default format to include commas in integer formatting, independent of localization.

Cleaned up tests.  All pass.

### CODE REVIEW

File: tests/test_tables.py [hunk 1]
```diff
@@ -267,8 +284,13 @@ def test_where_predicates(t):
     """)
 
 
-def test_sort(t):
+def test_sort(table):
+    t = table
+    t['totals'] = t['points'] * t['count']
+    print("test_sort")
```
Comment (adnanhemani) [inline tests/test_tables.py]: We should probably take away the print statements for the final push to master to ensure the code is clean.

File: tests/test_tables.py [hunk 2]
```diff
@@ -111,16 +110,19 @@ def test_column(table):
     assert_array_equal(table.column(1), np.array([9, 3, 3, 1]))
 
 
-def test_basic_points(t):
+def test_basic_points(table):
+    t = table
```
Comment (davidwagner) [inline tests/test_tables.py]: What's the purpose behind this change?

**Comment by deculler** [comment]
The use in the 't' and 'u' features is disallowed in the current version of
the unit test framework and provided no simplicity.  The feature needs to
be passed as a parameter to the 'test_...' function.  That 'table' where
't' was used and 'table2' where 'u' was.

Also there were sequential dependence on modifying the test case.  Now each
is standalone.

**Comment by davidwagner** [comment]
We merged #355, which is based on this.  Thanks very much, David!
